webauthn/helpers/mldsa.py

```diff
@@ -0,0 +1,31 @@
+from .cose import COSEAlgorithmIdentifier
+
+class ML_DSAPublicKey:
+    def __init__(self, alg, pub) -> None:
+        if not isML_DSA_available():
+            raise Exception("OQS Not installed")
```

Comment:
This should raise something new in `webauthn.helpers.exceptions` instead of a generic exception.